src/include/smmstore.h

```diff
@@ -21,6 +21,9 @@
 #define SMMSTORE_CMD_RAW_WRITE 6
 #define SMMSTORE_CMD_RAW_CLEAR 7
 
+/* Used by capsule updates */
```

Comment:
This could be extended to note that it's both a command and a modifier of other commands.

Reply:
https://github.com/Dasharo/coreboot/compare/5396e5c0e48880cec75b1046fef421e2889ca887..52ba34f9b8579bc8d7d31366ebd97a519c9975e5#diff-aa6ffd9e406ae20c5459b2ec33fbb8c4738db0c88b80493bdb9cd764e7618024R24